src/components/InputForm.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
```

Comment:
- [ ] It will be great to fix the errors instead of disabling the eslint. Remember the linters are there to help you catch code errors. Kindly make this fix and submit your project for another review.